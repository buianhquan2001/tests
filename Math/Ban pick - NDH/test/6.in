10
278 0.844
264 0.2933
300 0.1481
275 0.3668
254 0.3253
267 0.2935
276 0.2874
251 0.0625
297 0.576
260 0.5887
