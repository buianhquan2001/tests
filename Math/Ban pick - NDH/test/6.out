183
53
30
69
56
53
54
11
121
109
